src/tracer/MOM_tracer_flow_control.F90

```diff
@@ -217,12 +229,20 @@ subroutine call_tracer_register(HI, GV, US, param_file, CS, tr_Reg, restart_CS)
                  "If true, use the dyed_obc_tracer tracer package.", &
                  default=.false.)
 
+#ifndef _USE_MARBL_TRACERS
```

Comment:
These should now be dummy interfaces instead of CPP keys

Reply:
I'll email Bob (and cc you) in the next week or two, I may have misunderstood what he said on the Aug 24th telecon but late last year his suggestion was to create our own CPP key rather than overloading `USE_GENERIC_TRACERS`.

Reply:
So remember the call last year where Alistair said yes, hide it behind CPP keys. Turns out he changed his mind. If you look at how the generic tracer package is now implemented. There's a dummy interface that gets compiled every time.

Reply:
Definitely, don't overload `USE_GENERIC_TRACERS`, but you'll have to code up some extra stuff. Let me see if I can find the PR where he changed it.

Reply:
https://github.com/NOAA-GFDL/MOM6/commit/b1033f105376f03b76509e634e9314b1a05f4782#diff-49f1776f6843d1daf8d91a87aa1e3755